main.go

```diff
@@ -412,11 +457,12 @@ func getPipeline(ctx context.Context, pipelineName string, userProvidedNamespace
 			continue
 		} else if err == nil {
 			return pipeline, namespace, nil
-		} else if err != nil {
```

Comment:
🤨  Why this change? Do we not still need it?

Reply:
`err` is definitely `nil` when you reach this line and either static checks or `gopls` were hinting that. 🙂

Reply:
🤦  Oh yeah...